plugins/modules/system/parted.py

```diff
@@ -485,7 +485,7 @@ def parted_version():
     if len(lines) == 0:
         module.fail_json(msg="Failed to get parted version.", rc=0, out=out)
 
-    matches = re.search(r'^parted.+(\d+)\.(\d+)(?:\.(\d+))?$', lines[0])
+    matches = re.search(r'^parted.+(\d+)\.(\d+)(?:\.(\d+))?.?$', lines[0])
```

Comment:
How about:
```suggestion
    matches = re.search(r'^parted.+(\d+)\.(\d+)(?:\.(\d+))?', lines[0])
```

Reply:
better (correct) version:
```
    matches = re.search(r'parted.*\s+(\d+)\.(\d+)(?:\.(\d+))?', lines[0])
```
sample run:
```
...

parted_versions = [ """
parted (GNU parted) 3.3
Copyright (C) 2019 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by <http://git.debian.org/?p=parted/parted.git;a=blob_plain;f=AUTHORS>.
""", """
parted (GNU parted) 3.4.5
Copyright (C) 2019 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by <http://git.debian.org/?p=parted/parted.git;a=blob_plain;f=AUTHORS>.
""", """
parted (GNU parted) 3.3.14-dfc61
Copyright (C) 2019 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by <http://git.debian.org/?p=parted/parted.git;a=blob_plain;f=AUTHORS>.
"""]

for out in parted_versions:
"./parted-version2.py" 64L, 1928B written                                                                                        
[mator@archlinux ~]$ ./parted-version2.py   
(3, 3, 0)
(3, 4, 5)
(3, 3, 14)
```

Reply:
Splitting by spaces and using the last part will break if they ever decide to do things like `parted 1.2.3 special edition` :)